docker/airflow-python/Dockerfile

```diff
@@ -1,59 +1,16 @@
-ARG PYTHON_VERSION=3.6
-FROM python:${PYTHON_VERSION}-slim as airflow-base
+ARG AIRFLOW_VERSION=1.10.12
+ARG PYTHON_VERSION=3.7
+FROM apache/airflow:${AIRFLOW_VERSION}-python${PYTHON_VERSION}
 
-ENV AIRFLOW_BUILD_DEPS="freetds-dev python-dev libkrb5-dev libssl-dev libffi-dev libpq-dev git"
-ENV AIRFLOW_APT_DEPS="libsasl2-dev sasl2-bin libsasl2-2 libsasl2-dev libsasl2-modules freetds-bin build-essential default-libmysqlclient-dev apt-utils curl rsync netcat locales"
+USER root
 
-ENV AIRFLOW_VERSION=1.10.5
-ENV AIRFLOW_HOME /usr/local/airflow
-ENV AIRFLOW_GPL_UNIDECODE=yes
-ENV SLUGIFY_USES_TEXT_UNIDECODE=yes
+ARG AIRFLOW_VERSION=1.10.12
```

Comment:
the `AIRFLOW_VERSION` arg was already defined in line 1.

Reply:
Ja maar voor zover ik weet zijn die na de FROM regel niet meer beschikbaar en dien je ze dus opnieuw te definieren.